packages/react-form/src/validation/validators.ts

```diff
@@ -17,6 +17,12 @@ export function notEmptyString(error: ErrorContent<string>) {
   return validator(predicates.notEmptyString)(error);
 }
 
+export function positiveIntegerString(error: ErrorContent<string>) {
+  return validator(input => {
+    return input !== '' && (input.match(/[^0-9]/g) || []).length === 0;
```

Comment:
might make sense to move both this and `positiveNumericString` 's predicate into `predicates` and import them here :)

Reply:
hmm yea that's a good idea